setup/setup.ui

```diff
@@ -1335,6 +1347,142 @@
+                        <child>
+                          <object class="GtkComboBox" id="combobox_custom_theme">
+                            <property name="visible">True</property>
+                            <property name="can_focus">False</property>
+                            <property name="model">model_custom_theme</property>
+                            <property name="hexpand">True</property>
+                            <child>
+                              <object class="GtkCellRendererText" id="renderer4"/>
+                              <attributes>
+                                <attribute name="text">0</attribute>
+                              </attributes>
+                            </child>
+                          </object>
+                          <packing>
+                            <property name="expand">False</property>
+                            <property name="fill">False</property>
+                            <property name="position">1</property>
+                          </packing>
+                        </child>
+                      </object>
+                    </child>
+                    <child type="label">
+                      <object class="GtkLabel" id="label21">
+                        <property name="visible">True</property>
+                        <property name="can_focus">False</property>
+                        <property name="label" translatable="yes">&lt;b&gt;Theme&lt;/b&gt;</property>
+                        <property name="use_markup">True</property>
+                      </object>
+                    </child>
+                  </object>
+                  <packing>
+                    <property name="expand">False</property>
+                    <property name="fill">False</property>
+                    <property name="position">3</property>
+                  </packing>
+                </child>
+                <child>
+                  <object class="GtkFrame" id="frame7">
+                    <property name="visible">True</property>
+                    <property name="can_focus">False</property>
+                    <property name="label_xalign">0</property>
+                    <property name="shadow_type">none</property>
+                    <child>
+                      <object class="GtkBox" id="vbox12">
+                        <property name="orientation">vertical</property>
+                        <property name="visible">True</property>
+                        <property name="can_focus">False</property>
+                        <property name="spacing">6</property>
+                        <property name="margin_top">6</property>
+                        <property name="margin_start">24</property>
+                        <child>
+                          <object class="GtkCheckButton" id="checkbutton_custom_icon">
+                            <property name="label" translatable="yes">Use custom icon:</property>
```

Comment:
I wish to put GtkCheckButton and GtkComboBox at the same line likes:

"Use custom icon: [Adwaita]"